go.mod

```diff
@@ -49,35 +49,34 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/alertmanager v0.21.0
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.8.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.14.0
+	github.com/prometheus/common v0.15.0
```

Comment:
There's a promtool bugfix in here that you'll want in the changelog.

Reply:
Ok

Reply:
What is that fix? Not very obvious to me since `common` does not have a changelog.

Reply:
Detecting duplicate label names in exposition.